pkg/apis/roundtrip_shoot_migration_test.go

```diff
@@ -879,6 +879,7 @@ var _ = Describe("roundtripper cloudprofile migration", func() {
 						},
 						Workers: workerCIDR,
 					},
+					Zoned: true,
```

Comment:
Can you rather add a separate test for an Azure shoot that is zoned?

Also, can you add the service endpoints to the objects to see whether the conversion is working?

Reply:
yes I have separated zoned and non-zoned conversions in dedicated test cases. The service endpoint conversion is tested with the zoned test case.